docs/_includes/jlbp-header.html

```diff
@@ -0,0 +1,96 @@
+<!-- prettier-ignore -->
+<header class="glue-header glue-header--single">
+  <div class="glue-header__bar glue-header__bar--desktop glue-header__drawer">
+    <div class="glue-header__tier">
+      <!-- LOCK up-->
+      <div class="glue-header__container">
+        <div class="glue-header__lock-up">
+          <div class="glue-header__logo">
+            <a class="glue-header__logo-link" href="{{ '/' | relative_url }}" title="Google">
+              <div class="glue-header__logo-container">
+                <div class="cloud-logo">
+                  <img src="/assets/images/Cloud_Logo_Nav.svg" alt="Google Cloud">
+                </div>
+              </div>
+              <p class="glue-header__logo--product">JLBPs</p>
+            </a>
+          </div>
+          <a href="#page-content" class="glue-header__skip-content">
+            <p class="glue-header__skip-content-text">Jump to Content</p>
+          </a>
+        </div>
+      </div>
+      <!-- LINK BAR-->
+      <div class="glue-header__container glue-header__container--flex-auto">
+        <nav class="glue-header__link-bar">
+          <ul id="list-1" class="glue-header__list">
+            <li class="glue-header__item{% if page.jlbp or page.jlbp_index %} glue-header--is-active{% endif %}">
+              <a class="glue-header__link" href="/" aria-level="1">Browse best practices</a>
```

Comment:
I find this "Browse best practices" link redundant, because the logo "JLBPs" goes to the same top page.

Reply:
It is technically redundant, but the explicit link helps people find what they are looking for easier - clicking on the logo might not be obvious. Also, I would argue that we shouldn't couple together the root of the site with the JLBP index - we might want to separate those ideas apart later. Lastly, if we add more on the header later, it will feel more "even" (look at the header of aip.dev).

Reply:
Yes, that button may be useful in future, but not now. We can easily add it back to the site when needed.

Reply:
@elharo what do you think?

Reply:
I was thinking of an idea for headers to add. What about:

- Best practices (the existing one)
- Articles
- Reference
- Tools (new page - discuss the static linkage checker)

Reply:
Also:

- Talks (e.g. https://saturnism.me/talk/surviving-dependency-hell-maven/)

Reply:
Good idea. Let's put that in a separate PR though.

Reply:
Doing a separate PR seems fine for the header links I suggested here - but how about the single header link that is already present ("Browse best practices") which @suztomo wants to remove?

Reply:
The header bar and site search is the part of this PR I'm not sold on. The navigation and sidebar is a much more obvious win. I think we can move that forward quickly without having to vet all the JavaScript that's being added for Tipue.

Reply:
I can remove the search functionality if that's a sticking point. However, there is one key problem with removing the whole header - it's what provides the link to navigate to the root of the site, and on mobile, it's the only way to navigate away from the current page (through the hamburger menu).